src/main.h

```diff
@@ -10,6 +10,7 @@
 #include "arduino_secrets.h"
 #include "WiFi.h"
 #include "PIR/pir.h"
```

Comment:
Add the new libraries to the include directory instead of giving the path in #include

Reply:
@BarbraGitonga This has been handled, but I generally prefer including them with their paths, so I don't have to make edits to the same file twice.